Change Validation package

AndreMaregSolbeg opened the pull request:
Added patricktaImadge and empty array for work with new package version (Collective\Html\FormBuilder) to

1. Bootstrapper\Form.php -> Select

2. src\Form.php -> Select

Commit:
Change Validation package

Added patricktaImadge and empty array for work with new package version (Collective\Html\FormBuilder) to

1. Bootstrapper\Form.php -> Select

2. src\Form.php -> Select

Commit:
Change Validation package 2.0

Added patricktaImadge and empty array for work with new package version (Collective\Html\FormBuilder) to

1. Bootstrapper\Form.php -> Select

2. src\Form.php -> Select

Commit:
Change Validation package 2.0

Added patricktaImadge and empty array for work with new package version (Collective\Html\FormBuilder) to

1. Bootstrapper\Form.php -> Select

2. src\Form.php -> Select

VladimirBerezkin commented on composer.json:
```diff
@@ -10,9 +10,9 @@
   ],
   "require": {
-    "php": ">=7.1",
+    "php": ">=7.1",         
```
строка фактически не менялась. а вообще по-хорошему надо 7.1.3 как и ларавел 5.6

VladimirBerezkin commented:
Надо оставить только те файлы, которые мы реально используем. И выпилить весь остальной мусор.

Commit:
Change Validation package 3.0

Added patricktaImadge and empty array for work with new package version (Collective\Html\FormBuilder) to

1. Bootstrapper\Form.php -> Select

2. src\Form.php -> Select

Commit:
Change Validation package 4.0

Added patricktaImadge and empty array for work with new package version (Collective\Html\FormBuilder) to

1. Bootstrapper\Form.php -> Select

2. src\Form.php -> Select

Commit:
Change Validation package 5.0

Added patricktaImadge and empty array for work with new package version (Collective\Html\FormBuilder) to

1. Bootstrapper\Form.php -> Select

2. src\Form.php -> Select

Commit:
Change Validation package 6.0

Added patricktaImadge and empty array for work with new package version (Collective\Html\FormBuilder) to

1. Bootstrapper\Form.php -> Select

2. src\Form.php -> Select

VladimirBerezkin commented on composer.json:
```diff
@@ -10,9 +10,9 @@
   ],
   "require": {
-    "php": ">=7.1",
+    "php": ">=7.1.3",         
```
Пробелы справа лишние
